test/models/user_test.rb

```diff
@@ -3,7 +3,11 @@
 require 'test_helper'
 
 class UserTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  test 'should return email if name is blank, or name if name is present' do
```

Comment:
```suggestion
  test 'should return email if name is blank, or return name if name is present' do
```

なんとなくreturnを入れた方がいいような気がします

Reply:
ご指摘ありがとうございます🙇🏻